tests/clickhouse-test

```diff
@@ -959,6 +976,14 @@ def collect_sequential_list(skip_list_path):
 
 
 if __name__ == '__main__':
+    # Move to a new process group and kill it at exit so that we don't have any
+    # infinite tests processes left
+    # (new process group is required to avoid killing some parent processes)
+    os.setpgid(0, 0)
```

Comment:
This actually does not work for everything, since `timeout` also calls `setpgid`